lib/models/pr.js

```diff
@@ -153,70 +155,74 @@ class PR {
     get payload() {
         return this._payload;
     }
-        
+
     set payload(payload) {
         return this._payload = payload;
     }
 
     get installation() {
         return this._installation;
     }
-    
+
     get number() {
         return this._number;
     }
-    
+
     get owner() {
         return this._owner;
     }
-    
+
     get repo() {
         return this._repo;
     }
-    
+
     get id() {
         return this._id;
     }
-    
+
     get body() {
         return this._body = this._body || (this.payload.body || "").replace(/\r\n/g, "\n").trim();
     }
-    
+
     get processor() {
         return this.config.type.toLowerCase();
     }
-    
+
     get aws_s3_bucket() {
         if (process.env.ALLOW_MULTIPLE_AWS_BUCKETS == "no") {
             return process.env.AWS_BUCKET_NAME;
         }
         return this.owner == "whatwg" ? process.env.WHATWG_AWS_BUCKET_NAME : process.env.AWS_BUCKET_NAME;
     }
-    
+
     get isMultipage() {
         return !!this.config.multipage;
     }
 
     requiresPreview() {
         return !(/<!--\s*no preview\s*-->/.test(this.body));
     }
-    
+
+    async requestIncludes() {
+        this.includes = await scanIncludes(Head.fromPR(this).github_repo_url, this.config.src_file, this.config.type);
+    }
+
     touchesSrcFile() {
         let relFiles = this.relevantSrcFiles;
-        return this.files.some(f => relFiles.indexOf(f.filename) > -1);
+        return this.files.some(f => relFiles.has(f.filename));
     }
-    
+
     get relevantSrcFiles() {
-        return [this.config.src_file];
+        return this.includes || new Set([this.config.src_file]);
```

Comment:
What's the case when `this.includes` wouldn't be a `Set`?

Reply:
https://github.com/tobie/pr-preview/blob/master/test/models/pr.js#L99-L108 doesn't call `PR.init()`, or mock out the remote calls `.init()` would make, so we have to manually set up any state it needs.

If we take @sidvishnoi's suggestion in https://github.com/tobie/pr-preview/pull/60#issuecomment-628815148 to add a `hasIncludes` field to the config, that'll be another case where `this.includes` isn't set.